src/backend/marsha/core/api/video.py

```diff
@@ -113,7 +113,7 @@ def get_serializer_context(self):
         """Extra context provided to the serializer class."""
         context = super().get_serializer_context()
 
-        if self.request.resource:
+        if self.request and self.request.resource:
```

Comment:
Thanks, it was waiting somewhere in my git stash ^^

Reply:
There is something similar in the permissions that should be fix ? You had this error using the management command `generateschema` ?

Reply:
> There is something similar in the permissions that should be fixed ?

I don't think

> You had this error using the management command generateschema ?

Yes :)